backend.dev.Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM node:alpine3.17 AS builder
+WORKDIR /app
+
+COPY packages/server/package.json ./
+COPY yarn.lock ./
+RUN yarn remove node-cron @joystream/types
```

Comment:
Why is this needed?

Reply:
I think the build would fail with these included but I don't remember what was the exact issue.

Reply:
It didn't fail anymore ! I removed it now.